js/preload/preloadMain.js

```diff
@@ -64,6 +64,13 @@ const throttledSetBadgeCount = throttle(1000, function(count) {
     });
 });
 
+const throttledSetPresenceStatus = throttle(1000, function(status) {
```

Comment:
so, this is every second right? if so, can we look at increasing the period to 10 seconds?